frontend/src/components/user-management/student/SelectFormInputClassroom.tsx

```diff
@@ -1,23 +1,15 @@
 import React from "react";
 import { UseFormSetValue, UseFormWatch } from "react-hook-form";
-import {
-  GroupBase,
-  OptionBase,
-  Select,
-  SingleValue,
-} from "chakra-react-select";
+import { GroupBase, Select, SingleValue } from "chakra-react-select";
 
 import { ClassroomForm, ClassroomInput } from "../../../types/ClassroomTypes";
+import { MultiOption } from "../../../types/SelectInputTypes";
 
-interface Option extends OptionBase {
-  label: string;
-  value: string;
-}
 interface SelectFormInputClassroomProps {
   setValue: UseFormSetValue<ClassroomForm>;
   watch: UseFormWatch<ClassroomForm>;
   field: ClassroomInput;
-  options: Option[];
+  options: MultiOption[];
```

Comment:
@joyce-shi on the last PR we discussed why I added the MultiOption interface, it's for instance like this one where the value for an option can only be a string, and not a string | boolean, as is the case with Multi Option Select Menus.